mmfunctions

```diff
@@ -174,6 +175,15 @@ _premis_event_list(){
     fi
 }
 
+_generate_fingerprint(){
+    if [ ${PERCEPTUAL_HASH} = "1" ] ; then
+        SIGNATURE="$(basename ${INPUT%.*})_signature.xml"
+        ~/ffmpeg/ffmpeg -i "${INPUT}" -vf signature=format=xml:filename="${OUTDIR_INGESTFILE}/${MEDIAID}/metadata/fileMeta/objects/${SIGNATURE}" -map 0:v -f null -
```

Comment:
As the fingeprints are derived only from the input file as opposed to the derivatives or the package as a whole don't they belong with the other file specific data?  Wouldn't putting them in the package level metadata contradict the current structure of the packages and remove them from the context in which they were created?